app.py

```diff
@@ -536,13 +536,26 @@ def main():
                 col2.success("Post-Fire NBR Date 📅")
                 updated_date = col2.date_input("updated", datetime(2023, 7, 27), label_visibility="collapsed")
 
-                time_range = 7
-
-                # Process initial date
-                str_initial_start_date, str_initial_end_date = date_input_proc(initial_date, time_range)
-
-                # Process updated date
-                str_updated_start_date, str_updated_end_date = date_input_proc(updated_date, time_range)
+                min_date = date(2015, 6, 27) # sentinel-2 data initial date - 27th June, 2015
+                today = date.today()
+
+                # Error handler for date inputs
+                if initial_date < min_date or updated_date < min_date:
+                    st.error("Oops, sentinel-2 data starts from June 27, 2015. Please pick a later date.")
```

Comment:
It seems that this date corresponds to the launch of the Copernicus sentinel-2 mission rather than the gathering, ingestion, processing and availability of the data. But it'll do for now.